### PR TITLE
removed bug 'when authenticated we still can go to auth page '

### PR DESCRIPTION
**Pull request by NIveX2169**
Please do check routes !!

### CODE REVIEW

**Review by pooranjoyb — COMMENTED** [review]
Great job, @NIveX2169! 🎉 Thank you for submitting your pull request. Your contribution is valuable and we appreciate your efforts to improve our project.<p>We will promptly review your changes and offer feedback. Keep up the excellent work! Kindly remember to check our [contributing guidelines](https://github.com/pooranjoyb/popShop/blob/dev/CONTRIBUTING.md)</p>

**Comment by pooranjoyb** [comment]
None of the other routes seems to be working! Please check @NIveX2169 👀

**Comment by Mahitej28** [comment]
@NIveX2169 updates?

**Comment by NIveX2169** [comment]
> @NIveX2169 updates?
> 

I'm still working on it I need your help give me your discord

File: src/browserRoutes/routes/Routes.tsx [hunk 1]
```diff
@@ -5,46 +5,53 @@ import Home from "../../pages/Home/Home";
...
-        path: 'product',
-        element: <ProductDetail />,
-      },
-      {
-        path: 'cart',
-        element: <Cart />,
-      },
-      {
-        path: 'profile',
-        element: <Profile />
```
Comment (pooranjoyb) [inline src/browserRoutes/routes/Routes.tsx:39]: Seems like you forgot to add the `profile` route :/

**Review by pooranjoyb — CHANGES_REQUESTED** [review]

**Comment by Mahitej28** [comment]
@pooranjoyb have a look to it, seems good to me.

**Comment by pooranjoyb** [comment]
Awesome work @NIveX2169 🚀 
Sorry for the delay :/

**Pull request merged**